includes/class-wc-stripe-intent-controller.php

```diff
@@ -956,7 +956,7 @@ public function is_mandate_data_required( $selected_payment_type, $is_using_save
 	public function create_and_confirm_setup_intent( $payment_information ) {
 		$request = [
 			'payment_method'       => $payment_information['payment_method'],
-			'payment_method_types' => $payment_information['payment_method_types'],
+			'payment_method_types' => $payment_information['payment_method_types'] ?? [ $payment_information['selected_payment_type'] ],
```

Comment:
Hi @james-allan ! Your change [here](https://github.com/woocommerce/woocommerce-gateway-stripe/commit/3929b33faa6aedf7864b531e4ce845ebdaa9f186) broke this part when adding Cash App Pay tokens. I have added the previous version as a fallback to fix it.

Reply:
👍 Thanks for the heads up. I also changed it back to fallback to the `selected_payment_type` like you did. https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3311/commits/ebcd3b3604638c20673f6d59678159a0f780b86a.